app/views/layouts/_navbar.html.erb

```diff
@@ -60,17 +93,14 @@
           <li><%= link_to "<i class='material-icons left'>clear_all</i> Jobs".html_safe, [@course, :jobs], title: "View a history of autograding jobs" %></li>
 
           <% if @cud.instructor? %>
-            <li><%= link_to "<i class='material-icons left'>settings</i> Manage Course".html_safe, [:manage, @course], title: "Perform course-wise administrative actions" %></li>
+            <li><%= link_to "<i class='material-icons left'>settings</i> Course".html_safe, [:manage, @course], title: "Perform course-wise administrative actions" %></li>
           <% end %>
         <% end %>
 
-        <% if current_user && current_user.administrator? %>
-          <li><%= link_to "<i class='material-icons left'>lock</i> Manage Autolab".html_safe, admin_path, title: "Perform system-wide administrative actions" %></li>
-        <% end %>
-
         <% if user_signed_in? %>
+          <li><a class="dropdown-trigger" href="#!" data-target="dropdown2" data-beloworigin="true" data-hover="false"><i class='material-icons left'>lock</i>Manage Autolab<i class="material-icons right">arrow_drop_down</i></a></li>
```

Comment:
You are displaying this button even for non-admins. You should check that they are an admin first.